docs/debugging-testing/testing/unit_testing.md

```diff
@@ -30,7 +29,19 @@ Detailed instructions for supported operating systems are below.
    sudo easy_install pip
    ```
 
-1. Install Gcovr and [Mbed CLI](../build-tools/mbed-cli-1.html) with `pip install "gcovr>=4.1" mbed-cli`.
+### Installing dependencies on Debian or Ubuntu
+1. Install the compilation packages to the host:
+    ``
```

Comment:
@rajkan01 When you see code suggestions with two back ticks; it is because it is not possible to nest a set of three back ticks inside a Github suggestion and have it display correctly. It is up to you to replace the two back ticks with three back ticks.